### DISCUSSION TITLE
What version of onnxruntime is required by this project?

### DISCUSSION THREAD

**Discussion by wilderfield** [microsoft/onnxruntime-genai#634] [Q&A]
Do we need to build onnxruntime from main? Or can we use a release tag?

**Comment by baijumeswani** [comment]
You can do either. But using main is better since there are a few fixes in ort main that are not part of the ort release yet.

I recommend picking the [new onnxruntime-genai release](https://github.com/microsoft/onnxruntime-genai/releases/latest) and downloading the C API package from there.
If you want to build onnxruntime-genai from source, you can pick the onnxruntime dll/so from the C API package and build from source.
